rfcs/0036-rfc-process-team-amendment.md

```diff
@@ -0,0 +1,202 @@
+---
+feature: rfc-process-team-amendment
+start-date: 2018-10-27
+author: Robin Gloster <mail@glob.in>
+co-authors: Graham Christensen <graham@grahamc.com>
+related-issues: 1 (initial process), 24 (implementation)
+---
+
+# Summary
+[summary]: #summary
+
+This RFC proposes an RFC Steering Committee who decide on a group of RFC
+shepherds for each RFC who guide the discussion to a general consensus and then
+propose a motion for a "Final Comment Period" (FCP) with a disposition for
+acception, rejection or postponing. (see Glossary for a short definition of
+these terms)
```

Comment:
I think that «postponing» is never defined, and «FCP» and «disposition» are not mentioned in the Glossary/Terminology section.

Reply:
I think the idea of postponing is still unclear.

Maybe it would be better just to consider postponing a rejection, and say explicitly that new context or new tools or new knowledge can justify submitting a later RFC similar to an earlier rejected one?

Reply:
Even though the idea is more or less the same, I think separating postponement and rejection is useful to avoid strong wording, that can have negative effects on both sides of the discussion (people not wanting to use it and unconsciously preferring to let a RFC stall, people feeling rejected). IMO postponement is “we want that but are not ready for it” (eg. a RFC to make nixpkgs use `builtins.encryptString` everywhere), and rejection is “as of now, we don't think we want that” (eg. a RFC to remove overlays).

Now, agree with you that they are currently not well-defined-enough in the `Terminology` section and that it could use some wording to make it explicit that RFC choices can be revisited if the context|tools|knowledge changes.

Reply:
I am in favour of postponement (and of encouraging the shepherds to specify their personal — non-unanimous — expectations of what has to happen first for a real decision); but if there is a reason not to define it (or its consequences), I think omitting it might be still better than having an unclear hanging notion.